### DISCUSSION TITLE
Where should I report bugs after enabling experiments.css=true?

### DISCUSSION THREAD

**Discussion by dartess** [webpack/webpack#18534] [Q&A]
This is an experimental technology, and I don't understand whether I should report all the problems in the software, or maybe start a discussion?

A list of problems I encountered:

1. During the production build, `__webpack_modules__` has a broken key `44770;}`, so I get `TypeError: Cannot read properties of undefined (reading 'call') at __webpack_require__ ...`. I think it will be difficult to reproduce, since I cannot share the source codes, and since there is only one such key, I am afraid that if the code is cut, the case can disappear.

2. Inline assets (like background images in styles) are included in the build files twice: one `base64` is in the `.css`, one in the `.js`, although it is not used in the `.js`. This did not happen when using `mini-css-extract-plugin`; the background turned out only in the CSS file. This happens all the time and is easy to reproduce.

**Comment by alexander-akait** [comment]
> During the production build, __webpack_modules__ has a broken key 44770;}, so I get TypeError: Cannot read properties of undefined (reading 'call') at __webpack_require__ .... I think it will be difficult to reproduce, since I cannot share the source codes, and since there is only one such key, I am afraid that if the code is cut, the case can disappear.

Feel free to open an issue

> Inline assets (like background images in styles) are included in the build files twice: one base64 is in the .css, one in the .js, although it is not used in the .js.

Known problem and in roadmap, hope we will fix it soon

**Reply by dartess** [reply]
> Known problem and in roadmap, hope we will fix it soon

Is this publicly available? I couldn't find a relevant issue to follow.

**Reply by alexander-akait** [reply]
https://github.com/webpack/webpack/issues/14893

> url() in css leaves an asset JS Module in JS chunk
